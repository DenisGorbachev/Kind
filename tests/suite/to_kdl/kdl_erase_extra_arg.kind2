#kdl_erase = extra_arg
ErasedFun {
  45
}