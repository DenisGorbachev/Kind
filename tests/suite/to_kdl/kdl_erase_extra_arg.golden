You should not put arguments on the attribute 'kdl_erase'.
